File: app/src/main/java/com/dennis/saina/Question.java
package com.dennis.saina;

public class Question {
    private String mQuestionImage;
    private String mQuestionAnswer;
    private String mQuestionOption1;
    private String mQuestionOption2;

    public Question() {
    }

    public Question(String mQuestionImage, String mQuestionAnswer, String mQuestionOption1, String mQuestionOption2) {
        this.mQuestionImage = mQuestionImage;
        this.mQuestionAnswer = mQuestionAnswer;
        this.mQuestionOption1 = mQuestionOption1;
        this.mQuestionOption2 = mQuestionOption2;
    }

    public String getmQuestionImage() {
        return mQuestionImage;
    }

    public void setmQuestionImage(String mQuestionImage) {
        this.mQuestionImage = mQuestionImage;
    }

    public String getmQuestionAnswer() {
        return mQuestionAnswer;
    }

    public void setmQuestionAnswer(String mQuestionAnswer) {
        this.mQuestionAnswer = mQuestionAnswer;
    }

    public String getmQuestionOption1() {
        return mQuestionOption1;
    }

    public void setmQuestionOption1(String mQuestionOption1) {
        this.mQuestionOption1 = mQuestionOption1;
    }

    public String getmQuestionOption2() {
        return mQuestionOption2;
    }

    public void setmQuestionOption2(String mQuestionOption2) {
        this.mQuestionOption2 = mQuestionOption2;
    }
}

File: app/src/main/java/com/dennis/saina/Question.kt
package com.dennis.saina

open class Question {
    private var mQuestionImage: String? = null
    private var mQuestionAnswer: String? = null
    private var mQuestionOption1: String? = null
    private var mQuestionOption2: String? = null

    constructor() {}
    constructor(
        mQuestionImage: String?,
        mQuestionAnswer: String?,
        mQuestionOption1: String?,
        mQuestionOption2: String?
    ) {
        this.mQuestionImage = mQuestionImage
        this.mQuestionAnswer = mQuestionAnswer
        this.mQuestionOption1 = mQuestionOption1
        this.mQuestionOption2 = mQuestionOption2
    }

    fun getmQuestionImage(): String? {
        return mQuestionImage
    }

    fun setmQuestionImage(mQuestionImage: String?) {
        this.mQuestionImage = mQuestionImage
    }

    fun getmQuestionAnswer(): String? {
        return mQuestionAnswer
    }

    fun setmQuestionAnswer(mQuestionAnswer: String?) {
        this.mQuestionAnswer = mQuestionAnswer
    }

    fun getmQuestionOption1(): String? {
        return mQuestionOption1
    }

    fun setmQuestionOption1(mQuestionOption1: String?) {
        this.mQuestionOption1 = mQuestionOption1
    }

    fun getmQuestionOption2(): String? {
        return mQuestionOption2
    }

    fun setmQuestionOption2(mQuestionOption2: String?) {
        this.mQuestionOption2 = mQuestionOption2
    }
}